index.html

```diff
@@ -0,0 +1,142 @@
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@900&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/c90d8a2f33.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="about.css">
+</head>
+<body>
+  <header>
+    <nav class="social-head">
+      <ul class="nav-list">
+        <li><a target="_blank" rel="noopener" href="https://www.facebook.com/uonbi.ac.ke/"><i class="fa-brands fa-facebook-square"></i></a></li>
+        <li><a target="_blank" rel="noopener" href="https://twitter.com/uonbi?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor"><i class="fa-brands fa-twitter-square"></i></a></li>
+        <li><a href="#">Logout</a></li>
+        <li><a href="#">School Page</a></li>
+        <li><a class="engo" href="#">English</a></li>
+      </ul>
+    </nav>
+
+    <nav class="navbar">
+      <div class="ham" id="ham">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <a href="index.html" class="brand"><img src="./imgs/uonlogo.png" alt="uon logo"></a>
+      <ul class="navmenu" id="navmenu">
+          <li  class="items item1"><a href="about.html" class="nav-link about-link" id="about">About</a></li>
+          <li class="items item1"><a href="#programs" class="nav-link program-link">Program</a></li>
+          <li class="items item1"><a href="#guests" class="nav-link join-link">Speakers</a></li>
+          <li class="items"><a href="#sponsor" class="nav-link sponsor-link">Sponsor</a></li>
+          <li class="items"><a target="_blank" rel="noopener" href="https://www.uonbi.ac.ke/" class="nav-link school-link" id="school">UoNSchool</a></li>
+      </ul>
+
+    </nav>
+
+
+  </header>
+
+  <main>
+    <section class="first">
+      <div class="intro">
+        <h3>"Hello! It is Here Again"</h3>
+        <p class="title">NAIROBI INNOVATION WEEK EDITION 3 2022</p>
+        <p class="short-info">An Interactive session about innovation in Kenya and Africa brought to you by the University of Naorobi. This is to promote advanced changes in our continent.</p>
+        <p class="date">2022.04.26(WED) - 28(FRI)</p>
+        <p class="location">@ University of Nairobi, Great Court Main Campus.</p>
+      </div>
+      <div class="overle"></div>
+    </section>
+
+
+    <section class="second" id="programs">
+      <h2>Main Program</h2>
+      <hr>
```

Comment:
- [ ] [OPTIONAL] The color of the dividers is not the similar to the original design, please stick to the design as much as possible

|Your design|CC design|
|-|-|
|![Screenshot_20220528_052413](https://user-images.githubusercontent.com/25227022/170809915-25bc42e0-b4a3-4808-b6e0-08a49b92c2e2.png)|![Screenshot_20220528_052450](https://user-images.githubusercontent.com/25227022/170810024-cab463cf-140e-4da5-b7f6-3b392b0e795e.png)|
|![Screenshot_20220528_052424](https://user-images.githubusercontent.com/25227022/170809988-d99af6c8-dca4-4e7a-9041-4c0f4631d4f5.png)|![Screenshot_20220528_052504](https://user-images.githubusercontent.com/25227022/170809995-ec5dd711-9183-4ad5-9c60-1a317c7bb494.png)|